docs/documentation/quartz-3.x/tutorial/overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Library Overview
+---
+
+# The Quartz API
+
+The key interfaces and classes of the Quartz API are:
+
+
+| Type | |
+|-|--|
+| `IScheduler` | the main API for interacting with the scheduler |
+| `IJob` | an interface to be implemented by components that you wish to have executed by the scheduler |
+| `IJobDetail` | used to define instances of Jobs |
+| `ITrigger` | a component that defines the schedule upon which a given Job will be executed, job can have multiple associated triggers |
+| `JobBuilder` | used to define/build JobDetail instances, which define instances of Jobs |
+| `TriggerBuilder` | used to define/build Trigger instances |
+| `SchedulerBuilder` | used to define/build scheduler instances, requires Quartz 3.1 or later |
+
+In this tutorial for readability's sake following terms are used interchangeably: `IScheduler` and `Scheduler`, `IJob` and `Job`, `IJobDetail` and `JobDetail`, `ITrigger` and `Trigger`.
+
+A `Scheduler`'s life-cycle is bounded by it's creation, via a `SchedulerFactory` and a call to its `Shutdown()` method. 
+Once created the `IScheduler` interface can be used add, remove, and list Jobs and Triggers, and perform other scheduling-related operations (such as pausing a trigger). 
+However, the Scheduler will not actually act on any triggers (execute jobs) until it has been started with the `Start()` method, as shown in [Lesson 1](using-quartz.md).
+
+Quartz provides "builder" classes that define a Domain Specific Language (or DSL, also sometimes referred to as a "fluent interface"). In the previous lesson you saw an example of it, which we present a portion of here again:
```

Comment:
note: one thing that hasn't been very well documented is the new `SchedulerBuilder`, which is the backbone for DI configuration but also allows "magic string free configuration" for general usage.

Reply:
I've added a page, that we can use to start building something out. 👍